### DISCUSSION TITLE
Data output via MQTT

### DISCUSSION THREAD

**Discussion by janknieling** [SBFspot/SBFspot#603] [General]
Hi, 

Can anyone tell me if the data that I can sent via MQTT to Telegraf and put it then into an InfluxDB are the same or are the date that I can get via MQTT different from the data that SBFspot will put into the MySQL DB?

If there aren‘t any variables missing at MQTT I can use this as workaround as long as SBFspot doesn’t supports InfluxDB oder TimescaleDB as database backend

**Comment by janknieling** [comment]
@sbf- can you help me here? If SBFspot is delivering out the same data via MQTT as it writes into the MySQL DB the implementation of Postgres/TimescaleDB or InfluxDB isn‘t necessary. I think then it can be a solution to turn out the DB and only send the metrics via MQTT

**Reply by sbf-** [reply]
See [SBFspot.cfg](https://github.com/SBFspot/SBFspot/blob/master/SBFspot/SBFspot.cfg#L229)
Only spotdata (voltage, current, power, frequency and so on...)
No daydata (5 mins) nor monthdata (totals per month)

**Reply by janknieling** [reply]
Hmm that sounds good. I think the day and monthdata aren’t necessary because if SBFspot will pull the data every 5 mins from the inverters and publish these data via MQTT the day and monthdata can be generated from the database. InfluxDB for example has such a function
